src/main/index.ts

```diff
@@ -535,7 +540,12 @@ const getDevhubAppClient = async () => {
 
 const router = t.router({
   openSettings: t.procedure.mutation(() => {
-    PRIVILEDGED_LAUNCHER_WINDOWS[SETTINGS_WINDOW].show();
+    const mainWindow = PRIVILEDGED_LAUNCHER_WINDOWS[MAIN_WINDOW];
+    const [xMain, yMain] = mainWindow.getPosition();
+    const settingsWindow = PRIVILEDGED_LAUNCHER_WINDOWS[SETTINGS_WINDOW];
+    if (!settingsWindow) throw new Error('Settings window is undefined.');
```

Comment:
Interesting, the `settingsWindow` remains undefined until the `show` method is called right?

Reply:
No, it should already be defined here as it is being created as part of the `setupAppWindows` function (in this PR again, this was not the case in #201) .

Reply:
`.show()` only makes it be visible and come to front on the screen. The `BrowserWindow` javascript object is already defined before that.